src/app/guides/how-to-get-funding-for-your-open-source-projects/how-to-get-funding-for-your-open-source-projects.mdx

```diff
@@ -0,0 +1,62 @@
+## How To Get Funding For Your Open Source Projects
+
+# Overview
+
+As wonderful as open source is, one of the biggest challenges maintained projects face is finding funding, especially ones that are sustainable.
```

Comment:
[Proofreading] The _"ones"_ is a little ambiguous here. How about tightening up the sentence a bit?

```suggestion
As wonderful as open source is, one of the biggest challenges maintained projects can face is finding sustainable funding.
```